src/main/kotlin/com/openlattice/edm/processors/GetEntityTypeFromEntitySetEntryProcessor.kt

```diff
@@ -3,9 +3,10 @@ package com.openlattice.edm.processors
 import com.hazelcast.core.ReadOnly
 import com.kryptnostic.rhizome.hazelcast.processors.AbstractRhizomeEntryProcessor
 import com.openlattice.edm.EntitySet
+import com.openlattice.rhizome.hazelcast.entryprocessors.AbstractReadOnlyRhizomeEntryProcessor
 import java.util.*
 
-class GetEntityTypeFromEntitySetEntryProcessor : AbstractRhizomeEntryProcessor<UUID, EntitySet, Optional<UUID>>(), ReadOnly {
+class GetEntityTypeFromEntitySetEntryProcessor : AbstractReadOnlyRhizomeEntryProcessor<UUID, EntitySet, Optional<UUID>>() {
     override fun process(entry: MutableMap.MutableEntry<UUID, EntitySet?>): Optional<UUID> {
```

Comment:
Got this error when calling `EdmService.getPropertyTypesForEntitySet` :(
```2019-10-04T15:14:19,333 ERROR [qtp1475028309-91] com.openlattice.controllers.util.BaseExceptionHandler - 
com.hazelcast.nio.serialization.HazelcastSerializationException: Failed to serialize 'java.util.Optional'```

Reply:
Uhhhhh. What. Do we not have stream serializers for java.util.Optional?

Reply:
Well, we have a custom serializer for Optionals `OptionalStreamSerializers`, but " "Shouldn't Optional be Serializable?"  The short answer is that the Java Lambda (JSR-335) expert group considered and rejected it."

Reply:
wow. That is my least favorite news all year